charts/minibroker/values.yaml

```diff
@@ -13,10 +13,10 @@ tls:
 
 # The service broker server configuration.
 broker:
-  # The deployment configuration.
-  deployment:
+  # The service configuration.
+  service:
     # A port for the service broker HTTP API.
-    port: 8080
+    port: 80
```

Comment:
Not sure we'd ever need to change it, but that's okay, at least there's some plausible use.